website/docs/integrations/intellij.mdx

```diff
@@ -0,0 +1,148 @@
+---
+id: intellij
+title: JetBrains/IntelliJ Plugin - Manage your feature flags from JetBrains/IntelliJ IDEs
+description: ConfigCat IntelliJ IDEA Plugin. This is a step-by-step guide on how to use the ConfigCat JetBrains/IntelliJ  Plugin to manage feature flags in your IDEs.
+---
+
+// TODO jetbrains mp link
+
+<a href="https://marketplace.visualstudio.com/items?itemName=ConfigCat.configcat-feature-flags" target="_blank">ConfigCat's JetBrains/IntelliJ IDE plugin</a> to manage feature flags from IDEs.
+
+Connect your ConfigCat Product and Config to your JetBrains/IntelliJ IDE. Find your Feature Flag's usages in your code easily.
+
+This guide will help you install the JetBrains/IntelliJ IDE Plugin and familiarize you with its usage.
+
+## Feature overview
+
+- View your Feature Flags within the IDE.
+- Create Feature Flags within the IDE.
+- Open a Feature Flag on the ConfigCat Dashboard.
+- Find Feature Flag usages in your code.
+- Copy a Feature Flag's key to the clipboard.
+- View your Products & Configs.
+- Create Configs within the IDE.
+- Open a Config on ConfigCat Dashboard.
+
+<img src="/docs/assets/intellij/usage1.gif" className="zoomable" alt="Usage of ConfigCat Feature Flags & Settings Tab" />
+
+## Install plugin
+
+### Install within the IDE from Marketplace
+
+1. Open the `File` > `Settings/Preferences` menu.
+1. Go to the `Plugin` page and select the `Marketplace` tab.
+1. Search for `ConfigCat Feature Flags`.
+1. `Install` the plugin.
+1. Configure the plugin (see below).
+
+### Install manually within the IDE
+
+//TODO fix MP link
+
+1. Download the latest release from [GitHub](https://github.com/configcat/intellij-plugin/releases/latest) or from [JetBrains Marketplace](https://github.com/configcat/intellij-plugin/releases/latest)
```

Comment:
Itt szépen szerepel is a hol (a GitHubon), bár én még talán azt is ideírniám, hogy egy browserben.